rules.mk

```diff
@@ -123,6 +123,8 @@ ponyc_tag ?= wallaroolabs-$(strip $(latest_ponyc_tag))-release## tag for ponyc d
 ponyc_runner ?= wallaroolabs/ponyc## ponyc docker image to use
 debug ?= false## Use ponyc debug option (-d)
 debug_arg :=# Final argument string for debug option
+spike ?= false ## use ponycflags -D spike
```

Comment:
Maybe give a better description then `use ponycflags -D spike`? Possibly something like `Enable compile time network error testing (Spike)`?

Reply:
Ah, good idea! I was more or less mindlessly copying the form of the debug flag.